# Rosalind-Bioinfomatics-Stronghold
Solutions to Rosalind problems in R and python

List of problems:
| Problem | Solution |
| ------------- | ------------- |
| **[Counting DNA Nucleotides](https://rosalind.info/problems/dna/)**  | **[DNA.py](https://github.com/azizalazfar/Rosalind-Problems/blob/main/Python/DNA.py)**  |
| **[Transcribing DNA into RNA](https://rosalind.info/problems/rna/)**  | **[RNA.py]()** |
| **[Complementing a Strand of DNA](https://rosalind.info/problems/revc/)**  | **[REVC.py]()**  |
| **[Rabbits and Recurrence Relations](https://rosalind.info/problems/fib/)**  | **[FIB.py]()**  |
| **[	Computing GC Content](https://rosalind.info/problems/gc/)**  | **[GC.py]()**  |
| **[	Counting Point Mutations](https://rosalind.info/problems/hamm/)**  | **[HAMM.py]()**  |
| **[Mendel's First Law](https://rosalind.info/problems/iprb/)**  | **[IPRB.py]()**  |
| **[	Translating RNA into Protein](https://rosalind.info/problems/prot/)**  | **[PROT.py]()**  |
| **[Finding a Motif in DNA](https://rosalind.info/problems/subs/)**  | **[SUBS.py]()**  |
| **[Consensus and Profile](https://rosalind.info/problems/cons/)**  | **[CONS.py]()**  |
| **[	Overlap Graphs](https://rosalind.info/problems/grph/)**  | **[GRPH.py]()**  |
| **[Calculating Expected Offspring](https://rosalind.info/problems/iev/)**  | **[IEV.py]()**  |
| **[	Finding a Shared Motif](https://rosalind.info/problems/lcsm/)**  | **[LCSM.py]()**  |
| **[	Independent Alleles](https://rosalind.info/problems/lia/)**  | **[LIA.py]()**  |
| **[	Finding a Protein Motif](https://rosalind.info/problems/mprt/)**  | **[MPRT.py]()**  |
| **[	Inferring mRNA from Protein](https://rosalind.info/problems/mrna/)**  | **[MRNA.py]()**  |

| **[	Open Reading Frames](https://rosalind.info/problems/orf/)**  | **[ORF.py]()**  |
| **[	Enumerating Gene Orders](https://rosalind.info/problems/perm/)**  | **[PERM.py]()**  |
| **[	Calculating Protein Mass](https://rosalind.info/problems/prtm/)**  | **[PRTM.py]()**  |
| **[	Locating Restriction Sites](https://rosalind.info/problems/revp/)**  | **[REVP.py]()**  |

| **[	RNA Splicing](https://rosalind.info/problems/splc/)**  | **[SPLC.py]()**  |
| **[	Enumerating k-mers Lexicographically](https://rosalind.info/problems/lexf/)**  | **[LEXF.py]()**  |
| **[	Longest Increasing Subsequence](https://rosalind.info/problems/lgis/)**  | **[LGIS.py]()**  |
| **[	Genome Assembly as Shortest Superstring](https://rosalind.info/problems/long/)**  | **[LONG.py]()**  |

| **[	Perfect Matchings and RNA Secondary Structures](https://rosalind.info/problems/pmch/)**  | **[PMCH.py]()**  |
| **[	Partial Permutations](https://rosalind.info/problems/pper/)**  | **[PPER.py]()**  |
| **[Introduction to Random Strings](https://rosalind.info/problems/prob/)**  | **[PROB]()**  |
| **[	Enumerating Oriented Gene Orderings](https://rosalind.info/problems/sign/)**  | **[SIGN.py]()**  |
| **[Transitions and Transversions](https://rosalind.info/problems/tran/)**  | **[TRAN.py]()**  |
| **[	Completing a Tree](https://rosalind.info/problems/tree/)**  | **[TREE.py]()**  |
| **[	Error Correction in Reads](https://rosalind.info/problems/corr/)**  | **[CORR.py]()**  |
| **[	Counting Phylogenetic Ancestors](https://rosalind.info/problems/inod/)**  | **[INOD.py]()**  |
| **[k-Mer Composition](https://rosalind.info/problems/kmer/)**  | **[KMER.py]()**  |
| **[	Ordering Strings of Varying Length Lexicographically](https://rosalind.info/problems/lexv/)**  | **[LEXV.py]()**  |
| **[	Creating a Distance Matrix](https://rosalind.info/problems/pdst/)**  | **[PDST.py]()**  |
| **[Matching Random Motifs](https://rosalind.info/problems/rstr/)**  | **[RSTR.py]()**  |
| **[	Counting Subsets](https://rosalind.info/problems/sset/)**  | **[SSET.py]()**  |
| **[	Introduction to Alternative Splicing](https://rosalind.info/problems/aspc/)**  | **[ASPC.py]()**  |
| **[	Introduction to Set Operations](https://rosalind.info/problems/seto/)**  | **[SETO.py]()**  |
| **[	Constructing a De Bruijn Graph](https://rosalind.info/problems/dbru/)**  | **[DBRU.py]()**  |
| **[	Independent Segregation of Chromosomes](https://rosalind.info/problems/indc/)**  | **[INDC.py]()**  |
| **[	Mortal Fibonacci Rabbits](https://rosalind.info/problems/fibd/)**  | **[FIBD.py]()**  |
| **[Finding a Spliced Motif](https://rosalind.info/problems/sseq/)**  | **[SSEQ.py]()**  |
| **[	Catalan Numbers and RNA Secondary Structures](https://rosalind.info/problems/cat/)**  | **[CAT.py]()**  |
| **[	Speeding Up Motif Finding](https://rosalind.info/problems/kmp/)**  | **[KMP.py]()**  |
| **[Maximum Matchings and RNA Secondary Structures](https://rosalind.info/problems/mmch/)**  | **[MMCH.py]()**  |
| **[Reversal Distance](https://rosalind.info/problems/rear/)**  | **[REAR.py]()**  |
| **[	Expected Number of Restriction Sites](https://rosalind.info/problems/eval/)**  | **[EVAL.py]()**  |
| **[Distances in Trees](https://rosalind.info/problems/nwck/)**  | **[NWCK.py]()**  |






![](https://komarev.com/ghpvc/?username=azizalazfar)
| **[]()**  | **[]()**  |
